src/components/Drawer/index.jsx

```diff
@@ -77,17 +77,9 @@ export default function Drawer(props) {
       <RouterLink to="/runs" className={classes.drawerLink}>
         Runs
       </RouterLink>
-      <Divider />
-      <RouterLink to="/nodes?machine_type=smithi" className={classes.drawerLink}>
-        Nodes
+      <RouterLink to="/schedule" className={classes.drawerLink}>
+        Schedule
       </RouterLink>
-      <RouterLink to="/stats/nodes/lock" className={classes.drawerLink}>
-        Node Locks Stats
-      </RouterLink>
-      <RouterLink to="/stats/nodes/jobs" className={classes.drawerLink}>
-        Node Jobs Stats
-      </RouterLink>
-      <Divider />
```

Comment:
I think this was maybe removed accidentally?

Reply:
yep, let me add this back in my commit

Reply:
Just Added back and squashed the commit to the first commit. I removed the Divider, don't think we need that.

Reply:
Nice! We can also probably disable "Run" and "Dry Run" Buttons if the user is not logged in. Or show a warning on top of schedule page.